courseraresearchexports/sql/enrollments.sql

```diff
@@ -0,0 +1,99 @@
+/*
+enrollments
+An enrollment is a unique learner-course pair. Many tables log a learner's
+interactions in a course, and this view will aggregate key metrics for simple
+reporting purposes.
+
+Columns
+coursera_user_id
```

Comment:
are any of the below columns worth documenting? i.e what does `activity` mean? what does `completion` mean?